modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1047,6 +1051,15 @@ public AnchorPoint anchorPoint() {
         }
         return sf.anchorPoint(ff.literal(anchor.getX()), ff.literal(anchor.getY()));
     }
+    
+    /**
+     * 
+     * @return True if the layer has a text-max-angle explicitly provided.
+     */
+    public boolean hasTextMaxAngle() throws MBFormatException {
+        return parse.isPropertyDefined(layout, "text-max-angle");
+    }
```

Comment:
Maybe group this with wherever getTextMaxAngle is defined, just to make it easier to find related methods?

Also, does this need to be public? Probably better if it is protected or private.